frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -1,23 +1,580 @@
+        <small className="web-form-input">(children)</small>
+        <TextInput label="Education" dataKey="person.income.educationSelf" />
+        <small className="web-form-input">(self)</small>
+        <TextInput
+          label="Health care insurance"
+          dataKey="person.income.healthInsurance"
+        />
+        <TextInput
+          label="Health care expenses"
+          dataKey="person.income.healthExpenses"
+        />
+        <small className="web-form-input">
+          (excluding insurance listed above)
+        </small>
+        <TextInput
+          label="Other insurance"
+          dataKey="person.income.otherInsurance"
+        />
+        <small className="web-form-input">(describe)</small>
+        <TextInput
+          label="Entertainment"
+          dataKey="person.income.entertainment"
+        />
+        <TextInput
+          label="Laundry and dry cleaning"
+          dataKey="person.income.laundryDryClean"
+        />
+        <TextInput label="Donations" dataKey="person.income.donations" />
+        <TextInput label="Gifts" dataKey="person.income.gifts" />
+        <TextInput
+          label="Union and other dues"
+          dataKey="person.income.unionDues"
+        />
+        <TextInput
+          label="Garnishments or income withholding order"
+          dataKey="person.income.garnishments"
+        />
+        <TextInput
+          label="Retirement deposits"
+          dataKey="person.income.retireDeposits"
+        />
+        <small className="web-form-input">
+          (including pensions, 401(k), IRA, etc.)
+        </small>
+      </Section>
+
+      <Section name="8. Business Interest">
+        <CheckBox
+          label="I have no business interests"
+          dataKey="person.businessInterest"
+          name="businessInterests"
+        />
+        <CheckBox
+          label="I have the following business interests"
+          dataKey="person.noBusinessInterest"
+          name="businessInterests"
+        />
+      </Section>
+
+      <Section name="9. Financial Assets">
+        <CheckBox
+          label="I have no financial assets."
+          dataKey="person.financialAssets"
+          name="financialAssets"
+        />
+        <CheckBox
+          label="I have the following financial assets."
+          dataKey="person.noFinancialAssets"
+          name="financialAssets"
+        />
+      </Section>
+
+      <Section name="10. Real Estate">
+        <CheckBox
+          label="I have no real estate"
+          dataKey="person.realEstate"
+          name="realEstate"
+        />
+        <CheckBox
+          label="I have the following real estate"
+          dataKey="person.noRealEstate"
+          name="realEstate"
+        />
+      </Section>
+
+      <Section name="11. Personal Property">
+        <CheckBox
+          label="I have no personal property"
+          dataKey="person.personalProperty"
```

Comment:
same here about how it would be better to have one datakey and checkbox for a single boolean value, instead of two.